static/css/_forms.scss

```diff
@@ -119,7 +116,6 @@ form.new-post {
    	min-height: 20em;
    	overflow-y: hidden;
    	resize: none;
-   	box-sizing: content-box;
```

Comment:
I don't have the full history of the project so I don't know why you use a different box sizing here ?